docs/src/api/class-page.md

```diff
@@ -964,6 +964,58 @@ Optional event-specific initialization properties.
 ## async method: Page.dragAndDrop
 * since: v1.13
 
+This method drags the source element (or position) to the target element
```

Comment:
Drop `(or position)` it is always the elements matching the source/target selectors, or isn't it?